base/loading.jl

```diff
@@ -93,26 +103,24 @@ function find_in_path(name::AbstractString, wd = pwd())
     if wd !== nothing
         isfile_casesensitive(joinpath(wd,name)) && return joinpath(wd,name)
     end
-    for prefix in [Pkg.dir(); LOAD_PATH]
```

Comment:
@JeffBezanson compiling this `vcat` seems to be a difficult problem for inference to pre-generate, so i've rewritten in a way that generates less code (and thus saves a bit of time, multiplied by every precompiled module startup)